testplans/lotus-soup/go.mod

```diff
@@ -26,7 +26,7 @@ require (
 	github.com/ipfs/go-log/v2 v2.3.0
 	github.com/ipfs/go-merkledag v0.3.2
 	github.com/ipfs/go-unixfs v0.2.6
-	github.com/ipld/go-car v0.3.1-null-padded-files
+	github.com/ipld/go-car v0.3.2-0.20211001225732-32d0d9933823
```

Comment:
:rage:

Reply:
This is not a commit on master, and is not tagged.

Reply:
then ytf does it magically get changed i hate gomod idk how anything works is it my ide doing this??

Reply:
:shrug:? What go version?

Reply:
Ah, I see... It was already broken, the go mod just wasn't updated.

Reply:
what was broken? i honestly dunno how any of this works, i'm probably the one who broke it.

Reply:
Updated to a non-tagged commit not on master. It's not broken _yet_, but will be if the branch the commit lives on gets a force-push (which happens _frequently_ in this particular branch).